app/subscriber/src/components/tool-bar/AddToReportMenu.tsx

```diff
@@ -18,9 +18,7 @@ export const AddToReportMenu: React.FC<IAddToReportMenuProps> = ({ content }) =>
   const [activeReport, setActiveReport] = React.useState<IReportModel>();
   const [reportId, setReportId] = React.useState<number | null>(null);
   React.useEffect(() => {
-    if (!myReports.length) {
-      findMyReports().catch(() => {});
-    }
+    findMyReports().catch(() => {});
```

Comment:
We don't want to do this.  It's an expensive operation.
Something is updating the state and removing all the other reports.  We need to find what is causing that.